pkg/models/image.go

```diff
@@ -39,6 +39,21 @@ type ImageFilterType struct {
 	PerformerFavorite *bool `json:"performer_favorite"`
 	// Filter to only include images with these galleries
 	Galleries *MultiCriterionInput `json:"galleries"`
+	// Filter by url
+	URL *StringCriterionInput `json:"url"`
+}
+
+type ImageUpdateInput struct {
+	ClientMutationID *string  `json:"clientMutationId"`
+	ID               string   `json:"id"`
+	Title            *string  `json:"title"`
+	URL              *string  `json:"url"`
+	Date             *string  `json:"date"`
+	Rating           *int     `json:"rating"`
+	Organized        *bool    `json:"organized"`
+	StudioID         *string  `json:"studio_id"`
+	TagIds           []string `json:"tag_ids"`
+	PerformerIds     []string `json:"performer_ids"`
```

Comment:
You've defined `ImageUpdateInput` here, which means that references to `api.ImageUpdateInput` need to be updated to `models.ImageUpdateInput`.

This is why you're seeing the following errors:
```
D:\Projets\stashStar\internal\api\resolver_mutation_image.go:28:67: undeclared name: ImageUpdateInput
D:\Projets\stashStar\internal\api\resolver_mutation_image.go:46:71: undeclared name: ImageUpdateInput
D:\Projets\stashStar\internal\api\resolver_mutation_image.go:88:67: undeclared name: ImageUpdateInput
```

Also, you are missing the `GalleryIds` field. 

You can comment out this struct, rerun `make generate` and look for the `ImageUpdateInput` struct in `api/generated_models.go` for what it expects.

This explains this error:

```
D:\Projets\stashStar\internal\api\server.go:99:65: cannot use resolver (variable of type *Resolver) as ResolverRoot value in struct literal: *Resolver does not implement ResolverRoot (missing method ImageUpdateInput)
```